source/collections/VectorList.ooc

```diff
@@ -74,7 +74,7 @@ VectorList: class <T> {
 	}
 	pop: func -> T {
 		result := this[0]
-		this remove(0)
+		this removeAt(0)
```

Comment:
pop() leaked memory before since remove() allocates and returns a T object. Another solution is to use remove() and assign the returned object to result in pop().